src/Pond.tsx

```diff
@@ -32,12 +32,38 @@ type PondProps = {
   loadComponent?: JSX.Element
   /** Error callback the the pond is not able to reach actyxOS locally */
   onError?: (error: unknown) => void | JSX.Element
-  /** optional url to overwrite to local connection and connect to an other peer (default: `ws://localhost:4243/store_api`) */
-  url?: string
-  /** Hook, when the connection to the store is closed */
-  onStoreConnectionClosed?: () => void
-  /** Advanced configuration options for the Pond. */
-  pondOptions?: PondOptions
+  /**
+   * Optionally change the Actyx endpoint the Pond connects to
+   * and provide an error handler if the connection to Actyx is dropped.
+   * Defaults to `{}`
+   * */
+  connectionOpts?: ActyxOpts
+  /** 
+   * Advanced Pond configuration options.
+   * Defaults to `{}`
+   *  */
+  opts?: PondOptions
+  /**
+   * Manifest describing an Actyx application. Used for authorizing API access.
+   * Use `com.example.<somestring>` as `appId` for testing and development purposes,
+   * so you don't require a signed certificate.
+   * 
+   * Defaults to
+   * ```
+   * {
+   *   appId: 'com.example.react-pond-example',
+   *   displayName: 'React Pond Example',
+   *   version: '0.0.1'
+   * }
+   * ```
+   * */
+  manifest?: AppManifest
```

Comment:
I think we should not make this optional. 
This will eventually lead us into the same trouble as we had with the onError

Reply:
Well, it is optional but defaults to a `com.example....` manifest. I think that lowers the entrance barrier, don't you think?

Reply:
Ok, after thinking a bit on this: I agree and will make it required. 
Let's put the (little) pain on the start of the dev journey as opposed to when they want to release. Which could well happen, b/c there is no natural incentive to add the manifest once the app is running in dev...

Reply:
seconded